torch/csrc/jit/passes/canonicalize_ops.cpp

```diff
@@ -59,6 +59,7 @@ static void CanonicalizeOps(Block* block) {
 
       auto mm_result = mat1.mm(mat2);
       auto result = mat + mm_result;
+      ((Value*)result)->setType(it->output()->type());
```

Comment:
nit: don't use a C style cast. (Huh, you don't need to fix the type of `mm_result`?)